.github/workflows/rockspec.template

```diff
@@ -0,0 +1,35 @@
+local repo_url = '$repo_url'
```

Comment:
This template is used by the luarocks-tag-release action to generate release rockspecs that are then uploaded to luarocks.org.